docs/source/contrib.rst

```diff
@@ -32,17 +32,20 @@ environment dedicated to that extra test.
 Releasing and publishing a new package
 --------------------------------------
 
-Publishing to PyPI is done via a GitHub Actions workflow that is triggered when a tag is pushed.
+Publishing to PyPI is done via a GitHub Actions workflow that is triggered when a tag is pushed. Version is configured in the ``setup.cfg``. 
 
-First commit the version update to master and wait for tests to pass.
+The following process ensures correct version management: 
 
-You can push a tag by creating a `GitHub release <https://github.com/twisted/pydoctor/releases/new>`_.
+ - Create a branch: name it by the name of the new major ``pydoctor`` version, i.e. ``21.9.x``, re-use that same branch for bug-fixes.
```

Comment:
```suggestion
 - Create a branch: name it by the name of the new major ``pydoctor`` version, i.e. ``release-21.9.x``, re-use that same branch for bug-fixes.
```

If you want to reuse the branch and not delete it, it's best to not have it using the same convention as tag... to help with autocompletion.

I would say that is best to just name the branch `release-2.9.0` and delete the branch after the merge.

----------

when a bugfix is required you can create branch `release-21.9.1` based on the existing tag.

Reply:
I would prefer continue using the same branch name convention as the precedent versions. Actually I think it’s best to use the .x convention such that one can install a specific major version (with the latest fixes) with pip by selecting the branch name.

Reply:
ok. np.